scripts/unity/func_name_list.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+import sys
+import re
+import argparse
+
+def func_names_from_header(in_file, out_file):
+    with open(in_file) as f_in:
+        content = f_in.read()
+
+        with open(out_file,'w') as f_out:
+            x= re.findall(r"(struct\s|\s?)(\w+\s)(\*?)(\w+?)(\(.*?\))(;)", content, re.M | re.S)
+            for item in x:
+                f_out.write(item[3] + "\n")
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-i", "--input", type=str, help="input header file", required=True)
+    parser.add_argument("-o", "--output", type=str, help="output function list file")
+    parser.add_argument("-p", "--postfix", type=str, help="output file as input file with postfix")
+    args = parser.parse_args()
+
+    if args.output:
+        out_name = args.output
+    elif args.postfix:
+        out_name = args.input + args.postfix
+    else:
+        out_name = args.input + ".flist"
```

Comment:
This case is not needed and can be dropped, it is safe and clean to limit support to only args.output.